### DISCUSSION TITLE
Log4J logs not showing on k8s containers logs

### DISCUSSION THREAD

**Discussion by GalKarp** [Netflix/conductor#2329] [Q&A]
Hi, I see that part of the change to jdk11, you've decided to specify the log output folder to a non-standard one, is there any reason for that? one of the side effects of that is that we do not see logs in the /var/log/containers folder?

**Comment by aravindanr** [comment]
Conductor does not specify a log output folder. The default output is to the CONSOLE.

**Reply by GalKarp** [reply]
That was true before this change: 

https://github.com/Netflix/conductor/blob/15c0c144b00c0838cd961068f826f398b9b565de/docker/server/bin/startup.sh#L36

Are you going to fix it like before?

**Reply by apanicker-nflx** [reply]
The log4j file location was removed as part of the SpringBoot upgrade. Conductor no longer relies on log4j for logging but instead uses the logger supplied by the framework.
You could update the shell script above to accept a file location as an input and get back the behavior as before. Please feel free to open a PR with the change.

**Reply by maheshyaddanapudi** [reply]
You can try changing the startup command as below.

https://github.com/maheshyaddanapudi/conductor/blob/2668ecde1a181e7d41eb4b149784c9c9179e59ba/docker/server/bin/startup.sh#L36 

java ${JAVA_OPTS} -jar -DCONDUCTOR_CONFIG_FILE=$config_file conductor-server-*-boot.jar | tee /app/logs/server.log

This way the logs are also seen in docker container console output.
